app/code/Magento/InventoryConfigurableProductIndexer/etc/module.xml

```diff
@@ -9,6 +9,7 @@
     <module name="Magento_InventoryConfigurableProductIndexer" setup_version="1.0.0">
         <sequence>
             <module name="Magento_InventoryConfigurableProduct"/>
+            <module name="Magento_Catalog"/>
```

Comment:
Why do we need sequence `Magento_Catalog`?
We don't override any configuration, don't have a dependency on db tables from catalog during installation